### PR TITLE
Feature/wallet UI

### PR DESCRIPTION
**Pull request by jalarg**
# Summary
- Add wallet UI for the user
- Add cypress tests to wallet UI

# Details
- The wallet UI enhancement includes a user-friendly interface for managing account and transactions, providing a seamless experience for users.

# Evidence 

## Stellar Wallet: 

wallet-UI

tests

### CODE REVIEW

File: src/pages/wallet/index.tsx [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+      <div className="wallet-container flex flex-col min-h-screen">
+        <Navbar />
+        <div className="flex flex-col justify-start items-center gap-5 flex-grow py-3">
+          <div className="flex justify-evenly w-full gap-5 py-5 px-10">
+            <div className="flex flex-col items-center justify-center ">
+              <p className="wallet-balance-title text-sm sm:text-2xl">
+                Your balance
+              </p>
+              <p className="wallet-balance-amount text-sm sm:text-2xl font-bold">
+                0 Lumens (XML)
```
Comment (Genebson) [inline src/pages/wallet/index.tsx]: Where are we gonna show the user their balance after funding them?

File: src/pages/wallet/index.tsx [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+          </div>
+          <div className="bg-neutralMain flex flex-col px-[12.5%] text-black bg-blue-200 rounded-md border-2 border-blue-300 sm:h-[70vh] h-[70vh] sm:w-[60%]  hover:bg-blue-100">
+            <div className="flex items-center justify-center font-semibold gap-x-3">
+              <div className="w-[80%]">
+                <div className="wallet-publicKey-title flex items-start justify-start text-xs sm:text-xl font-semibold py-5">
+                  Your Stellar Public Key
+                </div>
+                <Input
+                  className="wallet-input text-sm font-semibold h-10"
+                  placeholder="GBXECMFOPNJQXSYHYT5VYI43RHKGPTSD3QVOTY7BY4GC7TDDXKGJUAXB"
```
Comment (Genebson) [inline src/pages/wallet/index.tsx]: This is hardcoded, this should be dynamic

File: src/pages/wallet/index.tsx [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+            </div>
+            <div className="flex items-center justify-center border-2 border-orange-300 rounded-md bg-orange-100 mt-3 p-3 gap-2">
+              <WarningOutlined className="wallet-warning-icon text-red-600 text-sm sm:text-2xl" />
+
+              <p className="wallet-warning-text text-sm sm:text-xl font-semibold text-center">
+                This account is currently inactive. to send 10,000 test lumen
+                (XLM) to the Stellar public key displayed above.
+              </p>
+              <button
+                className="button-addLumens activate-button font-extrabold text-blue-500 hover:text-blue-600"
```
Comment (Genebson) [inline src/pages/wallet/index.tsx]: `kebab-case`, please.

**Review by Genebson — APPROVED** [review]

File: src/pages/wallet/index.tsx (hunk 1 above)
Comment (jalarg) [reply src/pages/wallet/index.tsx]: Solved in commit 7c074238754049be79477222701d97713ceb761b.

File: src/pages/wallet/index.tsx (hunk 2 above)
Comment (jalarg) [reply src/pages/wallet/index.tsx]: Solved in commit 7c074238754049be79477222701d97713ceb761b.

File: src/pages/wallet/index.tsx (hunk 3 above)
Comment (jalarg) [reply src/pages/wallet/index.tsx]: fixed in 3a0673631727ad0ab816de0d6c7fa9288c78d37e.

File: cypress/e2e/home.cy.ts [hunk 4]
```diff
@@ -122,3 +122,69 @@ describe(" Footer: UI and functionality:", () => {
       .contains("Rocket wallet 2023");
   });
 });
+
+describe(" Wallet: UI and functionality:", () => {
```
Comment (Genebson) [inline cypress/e2e/home.cy.ts:125]: There's a blank space here.

File: src/pages/wallet/index.tsx [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+                secondary
+              >
+                Receive
+              </Button>
+            </div>
+          </div>
+          <div className="bg-neutralMain flex flex-col px-[12.5%] text-black bg-blue-200 rounded-md border-2 border-blue-300 sm:h-[70vh] h-[70vh] sm:w-[60%]  hover:bg-blue-100">
+            <div className="flex items-center justify-center font-semibold gap-x-3">
+              <div className="w-[80%]">
+                <div className="wallet-publicKey-title flex items-start justify-start text-xs sm:text-xl font-semibold py-5">
```
Comment (Genebson) [inline src/pages/wallet/index.tsx]:
```suggestion
                <div className="wallet-public-key-title flex items-start justify-start text-xs sm:text-xl font-semibold py-5">
```

**Review by Genebson — APPROVED** [review]

File: src/pages/wallet/index.tsx (hunk 5 above)
Comment (jalarg) [reply src/pages/wallet/index.tsx]: solved in commit 511e7114d29729205dbe09561458400491ef7083.

File: cypress/e2e/home.cy.ts (hunk 4 above)
Comment (jalarg) [reply cypress/e2e/home.cy.ts:125]: solved in commit 511e7114d29729205dbe09561458400491ef7083.

**Pull request merged**